vllm/transformers_utils/config.py

```diff
@@ -0,0 +1,15 @@
+from transformers import AutoConfig, PretrainedConfig
+
+from vllm.transformers_utils.configs import *  # pylint: disable=wildcard-import
+
+_CONFIG_REGISTRY = {
+    "mpt": MPTConfig,
+}
+
+
+def get_config(model: str) -> PretrainedConfig:
+    config = AutoConfig.from_pretrained(model, trust_remote_code=True)
+    if config.model_type in _CONFIG_REGISTRY:
+        config_class = _CONFIG_REGISTRY[config.model_type]
+        config = config_class.from_pretrained(model)
+    return config
```

Comment:
Oh I mean something like this:
```suggestion
def get_config(model: str) -> PretrainedConfig:
    if config.model_type in _CONFIG_REGISTRY:
        config_class = _CONFIG_REGISTRY[config.model_type]
        config = config_class.from_pretrained(model)
    else:
        config = AutoConfig.from_pretrained(model, trust_remote_code=True)
    return config
```

Reply:
Oh yeah, the problem of that is we can't get the initial `config` if the model is not an official HF model (like MPT, Falcon).

Reply:
oh oh i see. thanks for the clarification!